models/fal_clones_model.sql

```diff
@@ -0,0 +1 @@
+SELECT * FROM `learning-project-305919.dbt_meder.fal_clones`
```

Comment:
Change these to be a dbt source instead: https://docs.getdbt.com/docs/building-a-dbt-project/using-sources

Reply:
Same for all other models.

Reply:
Ideally, change to specifying the columns instead of `SELECT *`.

Reply:
dbt source gave me errors and didn't work. This was just a quick fix. Will be having a more in depth look at it. For the specification of columns, I use those models for Lightdash. In fal before I write the data to source, and Lightdash looks at the models to get the data, and the schema to define the columns and metrics. So I have models that get all of the data for each table that I want to run metrics on, then specify them for Lightdash in the schema.